test/example_data.dart

```diff
@@ -24,6 +24,17 @@ const int futureZulipFeatureLevel = 9999;
 // Users and accounts.
 //
 
+/// A fresh user ID, from a random but always strictly increasing sequence.
+int _nextUserId() => (_lastUserId += 1 + Random().nextInt(100));
+int _lastUserId = 1000;
+
+/// Construct an example user.
+///
+/// If user ID `userId` is not given, it will be generated from a random
+/// but increasing sequence.
```

Comment:
Sure. If for something else we want to generate random distinct IDs without constraining the ordering, zulip-mobile has a handy example we can follow, implemented by Ray years ago.

(Rereading it, I guess it can't just be transcribed from JS to Dart, because it relies on JS's sparse arrays. But a sparse array is basically just a map with int keys, plus an int for the length.)